Source/WebKit/DerivedSources-input.xcfilelist

```diff
@@ -183,6 +183,8 @@ $(PROJECT_DIR)/Shared/Cocoa/WebCoreArgumentCodersCocoa.serialization.in
 $(PROJECT_DIR)/Shared/Databases/IndexedDB/WebIDBResult.serialization.in
 $(PROJECT_DIR)/Shared/DisplayListArgumentCoders.serialization.in
 $(PROJECT_DIR)/Shared/EditorState.serialization.in
+$(PROJECT_DIR)/Shared/Extensions/WebExtensionContextParameters.serialization.in
+$(PROJECT_DIR)/Shared/Extensions/WebExtensionControllerParameters.serialization.in
```

Comment:
Were these never here before? Was anything not working with these missing? Just out of curiousity.

Reply:
Looks like not! This is auto updated by the build, so there must have been a bug when I added the files before.